src/dom/dom.js

```diff
@@ -1270,45 +1926,86 @@ function createMedia(pInst, type, src, callback) {
+ * example, `['assets/topsecret.mp4', 'assets/topsecret.ogv', 'assets/topsecret.webm']`.
+ * This is useful for ensuring that the video can play across different browsers with
+ * different capabilities. See
+ * <a href='https://developer.mozilla.org/en-US/docs/Web/HTML/Supported_media_formats'>MDN</a>
+ * for more information about supported formats.
+ *
+ * The second parameter, `callback`, is optional. It's a function to call once
+ * the video is ready to play.
  *
  * @method createVideo
- * @param  {String|String[]} src path to a video file, or array of paths for
- *                             supporting different browsers
- * @param  {Function} [callback] callback function to be called upon
- *                             'canplaythrough' event fire, that is, when the
- *                             browser can play the media, and estimates that
- *                             enough data has been loaded to play the media
- *                             up to its end without having to stop for
- *                             further buffering of content
- * @return {p5.MediaElement}   pointer to video <a href="#/p5.MediaElement">p5.MediaElement</a>
+ * @param  {String|String[]} src path to a video file, or an array of paths for
+ *                               supporting different browsers.
+ * @param  {Function} [callback] function to call once the video is ready to play.
+ * @return {p5.MediaElement}   new <a href="#/p5.MediaElement">p5.MediaElement</a> object.
  * @example
- * <div><code>
- * let vid;
+ * <div class='notest'>
+ * <code>
+ * function setup() {
+ *   noCanvas();
+ *
+ *   // Load a video and add it to the page.
+ *   // This may not work in some browsers.
+ *   createVideo('assets/small.mp4');
+ *
+ *   describe('A video of a toy robot.');
+ * }
+ * </code>
+ * </div>
+ *
+ * <div class='notest'>
+ * <code>
+ * function setup() {
+ *   noCanvas();
+ *
+ *   // Load a video and add it to the page.
+ *   // Provide an array options for different file formats.
+ *   createVideo(
+ *     ['assets/small.mp4', 'assets/small.ogv', 'assets/small.webm']
+ *   );
+ *
+ *   describe('A video of a toy robot.');
+ * }
+ * </code>
+ * </div>
+ *
+ * <div class='notest'>
+ * <code>
+ * let video;
+ *
  * function setup() {
  *   noCanvas();
  *
- *   vid = createVideo(
+ *   // Load a video and add it to the page.
+ *   // Provide an array options for different file formats.
+ *   // Use a callback function to set up the video once
+ *   // it's loaded.
+ *   video = createVideo(
  *     ['assets/small.mp4', 'assets/small.ogv', 'assets/small.webm'],
- *     vidLoad
+ *     handleVideo
  *   );
  *
- *   vid.size(100, 100);
+ *   describe('A video of a toy robot that plays in a loop.');
  * }
  *
- * // This function is called when the video loads
- * function vidLoad() {
- *   vid.loop();
- *   vid.volume(0);
+ * function handleVideo() {
+ *   video.size(100, 100);
+ *   video.loop();
+ *   video.volume(0);
```

Comment:
If I go directly to https://computiful.dev/sod-dom-functions/reference/#/p5/createVideo to simulate coming from a google search (or search for `createVideo`, go to the docs, then reload) the video doesn't immediately play, I suspect this is because we're muting after looping:
<img width="1030" alt="image" src="https://github.com/processing/p5.js/assets/5315059/170e69aa-0692-4620-980b-a5d4f2996822">

Maybe we should try muting first? If that does indeed fix it, it's probably worth mentioning explicitly in the docs that that ordering matters because of how the browser deals with autoplaying media.